vms/proposervm/vm.go

```diff
@@ -333,7 +333,34 @@ func (vm *VM) SetPreference(ctx context.Context, preferred ids.ID) error {
 		return err
 	}
 
-	// reset scheduler
+	var (
+		nextStartTime   time.Time
+		parentTimestamp = blk.Timestamp()
+	)
+	if vm.IsDurangoActivated(parentTimestamp) {
+		nextStartTime, err = vm.resetPostDurangoScheduler(ctx, blk, pChainHeight)
+	} else {
+		nextStartTime, err = vm.resetPreDurangoScheduler(ctx, blk, pChainHeight)
+	}
+	if err != nil {
+		return nil
+	}
```

Comment:
Don't we still need to reset the scheduler post durango? Otherwise nodes may just never update the scheduler - right?